### PR TITLE
Bring back downloads

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

**Review by TayouVR — APPROVED** [review]

**Review by flowln — COMMENTED** [review]
image
_oh no_

File: src/download/linux.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+```
+
+You can replace yay -S with your preferred [AUR helper's](https://wiki.archlinux.org/title/AUR_helpers) install command.
+</div>
+
+<div class="infobox top">
+
+# <img src="https://www.vectorlogo.zone/logos/centos/centos-icon.svg" height="20"> CentOS Stream / Red Hat Enterprise Linux
+
+Two RPM packages are available on [Copr](https://copr.fedorainfracloud.org/coprs/sentry/prismlauncher/).
```
Comment (jopejoe1) [inline src/download/linux.md:64]: Packeged by a difrent person now
```suggestion
Two RPM packages are available on [Copr](https://copr.fedorainfracloud.org/coprs/g3tchoo/prismlauncher/).
```

File: src/download/linux.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+```bash
+# load in system information
+source /etc/os-release
+# enables the copr repo
+sudo dnf copr enable sentry/prismlauncher epel-${VERSION}-$(uname -m)
```
Comment (jopejoe1) [inline src/download/linux.md:70]: Same as above
```suggestion
sudo dnf copr enable g3tchoo/prismlauncher epel-${VERSION}-$(uname -m)
```

File: src/download/linux.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+# stable releases
+sudo dnf install prismlauncher
+# nightly releases (git master)
+sudo dnf install prismlauncher-nightly
```
Comment (jopejoe1) [inline src/download/linux.md:74]: Nightly version is currently not included in copr

File: src/download/linux.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+You can replace una install with your preferred [MPR helper's](https://docs.makedeb.org/using-the-mpr/list-of-mpr-helpers/) install command.
+</div>
+
+<!--
+<div class="infobox top">
+
+# <img src="https://www.vectorlogo.zone/logos/getfedora/getfedora-icon.svg" height="20"> Fedora
+
+Two RPM packages are available on [Copr](https://copr.fedorainfracloud.org/coprs/sentry/prismlauncher/).
```
Comment (jopejoe1) [inline src/download/linux.md:130]: Maintainer changed
```suggestion
Two RPM packages are available on [Copr](https://copr.fedorainfracloud.org/coprs/g3tchoo/prismlauncher/).
```

File: src/download/linux.md [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+```bash
+# enables the copr repo
+sudo dnf copr enable sentry/prismlauncher
```
Comment (jopejoe1) [inline src/download/linux.md:134]: Same as above
```suggestion
sudo dnf copr enable g3tchoo/prismlauncher
```

File: src/download/linux.md [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+# stable releases
+sudo dnf install prismlauncher
+# nightly releases (git master)
+sudo dnf install prismlauncher-nightly
```
Comment (jopejoe1) [inline src/download/linux.md:138]: Nightly is currently not included in copr

File: src/download/linux.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+```
+
+</div>
+<div class="infobox top">
+
+# <img src="https://www.gentoo.org/assets/img/logo/gentoo-signet.svg" height="20" /> Gentoo
+
+Ebuilds are available in the official Gentoo repository, under [`games-action/prismlauncher`](https://packages.gentoo.org/packages/games-action/prismlauncher).
```
Comment (jopejoe1) [inline src/download/linux.md:146]: Not in the official gentoon repo currently

File: src/download/linux.md [hunk 8]
```diff
@@ -0,0 +1,191 @@
...
+
+A [Nix derivation](https://github.com/PrismLauncher/PrismLauncher/blob/develop/nix/NIX.md) is available.
+</div>
+
+<!--
+<div class="infobox top">
+
+# <img src="https://upload.wikimedia.org/wikipedia/commons/d/d0/OpenSUSE_Logo.svg" height="20"> openSuse
+
+Prism Launcher is available on the [game:tools](https://software.opensuse.org/download.html?project=games%3Atools&package=prismlauncher) community repository.
```
Comment (jopejoe1) [inline src/download/linux.md:176]: Currently not included in the offical repo, but is avaible in the same copr repo as the fedora package

File: src/download/mac.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+      {% image "Modpack Installer", "./src/img/screenshots/mac/LauncherLight.png", "./src/img/screenshots/mac/LauncherDark.png" %}
+    </div>
+  </div>
+</div>
+
+<div class="infobox top">
+
+## Advanced MacOS install options
+
+### Homebrew Package (community-maintained)
```
Comment (jopejoe1) [inline src/download/mac.md:29]: Currently not in homebrew

File: src/download/linux.md [hunk 10]
```diff
@@ -0,0 +1,191 @@
...
+
+</div>
+
+<div class="infobox top">
+
+# <img src="https://bitcu.co/en/wp-content/uploads/2020/07/Void_Linux_logo.svg_.png" height="20"> Void Linux
+
+Prism Launcher is available on the official Void repository.
```
Comment (jopejoe1) [inline src/download/linux.md:184]: Currently not included in the offical repo

File: src/download/mac.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+#### Installation instructions
+
+```bash
+brew install --cask prism
+```
+
+<!--
+### MacPorts Package (community-maintained)
```
Comment (jopejoe1) [inline src/download/mac.md:38]: Currently not in macports

File: src/download/windows.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+
+### CLI (community-maintained)
+
+#### Winget Package
+
+```powershell
+winget install -e PrismLauncher.PrismLauncher
+```
+
+#### Scoop Package
```
Comment (jopejoe1) [inline src/download/windows.md:48]: Currently not in scoop

File: src/download/windows.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+```powershell
+scoop bucket add games
+scoop install prismlauncher 
+scoop install prismlauncher-qt5 #if you want Windows-Legacy instead
+```
+
+#### Chocolatey Package
```
Comment (jopejoe1) [inline src/download/windows.md:56]: Currently not chocolatey

File: src/download/windows.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+- [Legacy portable (.zip)](https://github.com/PrismLauncher/PrismLauncher/releases/download/{{version.current}}/PrismLauncher-Windows-Legacy-Portable-{{version.current}}.zip)
+
+## Advanced Windows install options
+
+- [System Install (.zip)](https://github.com/PrismLauncher/PrismLauncher/releases/download/{{version.current}}/PrismLauncher-Windows-{{version.current}}.zip)
+- [Legacy system install (.zip)](https://github.com/PrismLauncher/PrismLauncher/releases/download/{{version.current}}/PrismLauncher-Windows-Legacy-{{version.current}}.zip)
+
+### CLI (community-maintained)
+
+#### Winget Package
```
Comment (jopejoe1) [inline src/download/windows.md:42]: Currently not in winget

File: src/download/linux.md [hunk 15]
```diff
@@ -0,0 +1,191 @@
...
+  order: 3
+---
+
+<div class="download-content">
+  <div class="row">
+    <div class="column">
+      <div>
+        <h1>Linux Download</h1>
+        <br>
+        <a class="button is-big" href="https://flathub.org/apps/details/org.prismlauncher.PrismLauncher">Install from FlatHub</a>
```
Comment (jopejoe1) [inline src/download/linux.md:14]: Not in flathub yet

**Review by jopejoe1 — COMMENTED** [review]
Checked wich of the listed download methods still work.

File: src/download/windows.md (hunk 12 above)
Comment (TayouVR) [reply src/download/windows.md:48]: its in scoop

Comment (jopejoe1) [reply src/download/windows.md:48]: Missed it as i was searching for it

**Pull request merged**